akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/StubbedActorContext.scala

```diff
@@ -166,12 +166,14 @@ private[akka] final class FunctionRef[-T](override val path: ActorPath, send: (T
     val i = new BehaviorTestKitImpl[U](system, p, BehaviorImpl.ignore)
     _children += p.name -> i
 
-    new FunctionRef[U](p, (message, _) => {
-      val m = f(message);
-      if (m != null) {
-        selfInbox.ref ! m; i.selfInbox().ref ! message
-      }
-    })
+    new FunctionRef[U](
+      p,
+      (message, _) => {
```

Comment:
keep the `p` and `(message，_)` in the same line?

Reply:
Given that it splitting up  (p, <long function block>), the change the formatter made seems fair